connector/microsoft/microsoft.go

```diff
@@ -92,6 +92,10 @@ func (c *microsoftConnector) oauth2Config(scopes connector.Scopes) *oauth2.Confi
 		microsoftScopes = append(microsoftScopes, scopeGroups)
 	}
 
+	if scopes.OfflineAccess {
+		microsoftScopes = append(microsoftScopes, "offline_access")
```

Comment:
This needs to be a package constant, like the scopes above. Also, please link to the microsoft docs in a code comment. Not every reviewer will be familiar with the Microsoft OAuth APIs 😃